RecoTracker/MkFit/test/dumpMkFitGeometry.py

```diff
@@ -1,9 +1,9 @@
 import FWCore.ParameterSet.Config as cms
 
 from Configuration.Eras.Era_Run3_cff import Run3
-from Configuration.ProcessModifiers.trackingMkFitDevel_cff import trackingMkFitDevel
+from Configuration.ProcessModifiers.trackingMkFit_cff import trackingMkFit
```

Comment:
this modifier doesn't exit in cmssw (was moved in https://github.com/cms-sw/cmssw/pull/38389)

this is why [the unit test is failing](https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-d391b8/28767/unitTests/src/RecoTracker/MkFit/test/testDumpMkFitGeometry/testing.log)

Reply:
Argh, apparently my base is too old ... and I thought I'm fixing it. Sorry for the mess, I said I suck at this :)

Reply:
But wait ... if mkFit is loaded by default now, we don't need these modifiers anymore at all, right? Anyway, I will just roll-back the Devel I took away and add a unit test for Phase2.

Reply:
if you use the `devel` one it will use mkfit for all the iteration, though I don't know what that implies in this particular file.

Reply:
On a cursory look, if the only point of this file is to run `DumpMkFitGeometry` and its dependencies (which is `MkFitGeometryESProducer`, which itself depends on
https://github.com/cms-sw/cmssw/blob/e01b0cd05b0622c49c8d306929c9b6b258943af0/RecoTracker/MkFit/plugins/MkFitGeometryESProducer.cc#L67-L69
I would think this configuration would not need any Era specification at all, since the behavior is fully driven by the geometry (from GT in Run 3, from XML file in Phase 2).

Reply:
>  I would think this configuration would not need any Era specification at all, since the behavior is fully driven by the geometry (from GT in Run 3, from XML file in Phase 2).

I agree.

Reply:
There was further discussion between @makortel and @slava77 and they concluded the way how these files look now is ok, that it is safer and clearer to have the era setup included.